lib/gc/Dialect/OneDNNGraph/OneDNNGraphOps.cpp

```diff
@@ -158,22 +122,22 @@ LogicalResult onednn_graph::MatMulOp::inferReturnTypeComponents(
     // Not supported
     return failure();
   }
-  auto getShapeIdx = [](ArrayRef<ShapeAdaptor> operands, size_t i) {
-    return operands[i];
-  };
   // final shape
   auto retShape = ShapedTypeComponents(outShape, lhsShape.getElementType());
   inferredReturnShapes.push_back(retShape);
   // check for bias broadcasting
   if (adaptor.getBias()) {
     ShapeAdaptor biasShape(adaptor.getBias().getType());
     ShapeAdaptor matShape(retShape);
+    llvm::SmallVector<int64_t> matmulShape;
+    matShape.getDims(matmulShape);
```

Comment:
Can we use something like `getShape` to get the ArrayRef of shape for performance?